src/Worker/ExpirePosts.php

```diff
@@ -52,7 +52,7 @@ public static function execute()
 			self::addMissingEntries();
 		}
 
-		// Set the expiry for origin posta
+		// Set the expiry for origin post
```

Comment:
I believe it's more general.
```suggestion
		// Set the expiry for origin posts
```